tock/tock/context_processors.py

```diff
@@ -0,0 +1,13 @@
+def version_url(request):
+    import re
+    from django.conf import settings
+    base_url = 'https://github.com/18F/tock/'
+    name = settings.VERSION
+    response = {
+        'release_url': '%stree/%s' % (base_url, name),
+        'release_name': settings.VERSION,
+    }
+    if re.match('/v20[1-9][0-9][0-9]+\.[0-9]+/', name):
+        response['release_url'] = '%sreleases/tag/%s' % (base_url, name)
+
+    return response
```

Comment:
This will return a `response` consisting of the following:

```py
# if $CIRCLE_TAG doesn't exist or is equal to anything other than a release tag.
response = {
    'release_url': 'https://github.com/18F/tock/tree/master',
    'release_name': 'master',
}

# if $CIRCLE_TAG meets our release tag convention from the CI/CD PR.
response = {
    'release_url': 'https://github.com/18F/tock/releases/tag/v20180207.1',
    'release_name': 'v20180207.1',
}
```